python/lsst/afw/table/functorKey.cc

```diff
@@ -22,16 +22,43 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/afw/coord/Coord.h"
+#include "lsst/afw/table/arrays.h"
+#include "lsst/afw/table/FunctorKey.h"
 
 namespace py = pybind11;
 
 using namespace lsst::afw::table;
 
+template <typename T>
+void declareFunctorKeys(py::module & mod, const std::string suffix){
+    py::class_<OutputFunctorKey<T>, std::shared_ptr<OutputFunctorKey<T>>>
+        clsOutputFunctorKey(mod, ("OutputFunctorKey"+suffix).c_str());
```

Comment:
I was a bit lazy, not wanting to worry about the size of the buffer when concatenating the suffix.